### PR TITLE
Fix promotion pipeline

### PR DESCRIPTION
**Pull request by epananth**
### To double check:

* [ ] The right tests are in and and the right validation has happened.  Guidance:  https://github.com/dotnet/core-eng/tree/master/Documentation/Validation
After my  https://github.com/dotnet/arcade/pull/7086/files pr went in, its failing promotion cos I removed download artifacts in yaml. Adding this back to fix promotion. I will create another PR to enable the new download after this.

### CODE REVIEW

File: eng/publishing/v3/publish-assets.yml [hunk 1]
```diff
@@ -31,6 +31,24 @@ jobs:
...
+      inputs:
+        buildType: specific
+        buildVersionToDownload: specific
+        project: $(AzDOProjectName)
+        pipeline: $(AzDOPipelineId)
+        buildId: $(AzDOBuildId)
+        downloadType: 'specific'
+        itemPattern: |
+          PackageArtifacts/**
+          BlobArtifacts/**
```
Comment (adiaaida) [inline eng/publishing/v3/publish-assets.yml:43]: why not just add the missing ones below and not have the enabled:false bit?

**Review by adiaaida — COMMENTED** [review]
You probably want to remove the UseStreamingPublishing:true on row 113 until arcade promotes to itself too

**Comment by mmitche** [comment]
I think I get it. This is just temporary to unblock the promotion pipeline until arcade gets a new version of publishing. Then this change will be reverted.

**Review by mmitche — APPROVED** [review]

**Review by adiaaida — APPROVED** [review]

**Comment by adiaaida** [comment]
Nevermind on that last comment. I realized that it'll just be ignored because current arcade doesn't know about it so it should ignore it.

**Comment by epananth** [comment]
I used a this branch to promote my changes to .Net Eng- Latest. Here is the promotion build -> https://dev.azure.com/dnceng/internal/_build/results?buildId=1083838&view=logs&s=380ab8c6-4139-55ff-15c4-fb48260f4dca, then ran trigger subscription -> https://github.com/dotnet/arcade/pull/7224#pullrequestreview-633881002.  

We no longer need this PR.

**Pull request closed without merging**